Sources/RestKit/Authentication/ConfigBasedAuthenticatorFactory.swift

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright IBM Corporation 2019
```

Comment:
This copyright didn't get updated.  I suppose that's a deficiency in my script.

Reply:
I can reformat this manually for now and look at the script later.